sdk.go

```diff
@@ -128,8 +130,10 @@ func (c *SDK) calculatePublicChannelTopicID(name string, symkey int) (topicID st
 
 func (c *SDK) call(cmd string, res interface{}) error {
```

Comment:
ok, so all calls to this method happen now from [dictionary.go](https://github.com/status-im/status-go-sdk/blob/master/dictionary.go#L71) they're centralized and isolated, so no need to extract them on client.go and clientwrapper.go.

All of them are using a [shhRequest struct](https://github.com/status-im/status-go-sdk/blob/master/dictionary.go#L8) to compose messages then they marshal it, and the result is sent to this call method.

My proposal here is to change `call` to accept this `shhRequest` struct instead of a string (that will allow us removing all json.Marshall around dictionary.go). Additionally making `ShhRequest` public, so, this method will look like:

```go
func (c *SDK) call(req ShhRequest, res interface{}) error {
	log.Println("[ REQUEST ] : " + cmd)
        body, err := c.rpcClient.Call(req)
	if err != nil {
		return err
	}
	log.Println("[ RESPONSE ] : " + body.(string))

	return json.Unmarshal([]byte(body.(string)), &res)
}
```